zetaclient/tss_signer.go

```diff
@@ -12,25 +12,30 @@ import (
 	"strings"
 	"time"
 
-	"github.com/binance-chain/tss-lib/ecdsa/keygen"
+	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
+
+	"github.com/zeta-chain/zetacore/zetaclient/metrics"
+
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/zeta-chain/go-tss/p2p"
+	"github.com/zeta-chain/zetacore/common"
+	"github.com/zeta-chain/zetacore/x/crosschain/types"
+	"github.com/zeta-chain/zetacore/zetaclient/config"
+
 	"github.com/btcsuite/btcutil"
 	ethcommon "github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
 	gopeer "github.com/libp2p/go-libp2p/core/peer"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 	tmcrypto "github.com/tendermint/tendermint/crypto"
-	"github.com/zeta-chain/zetacore/common"
+
+	thorcommon "github.com/zeta-chain/go-tss/common"
+	"github.com/zeta-chain/tss-lib/ecdsa/keygen"
 	zcommon "github.com/zeta-chain/zetacore/common/cosmos"
-	"github.com/zeta-chain/zetacore/x/crosschain/types"
-	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
-	"github.com/zeta-chain/zetacore/zetaclient/config"
-	"github.com/zeta-chain/zetacore/zetaclient/metrics"
-	thorcommon "gitlab.com/thorchain/tss/go-tss/common"
-	"gitlab.com/thorchain/tss/go-tss/keysign"
-	"gitlab.com/thorchain/tss/go-tss/p2p"
-	"gitlab.com/thorchain/tss/go-tss/tss"
+
+	"github.com/zeta-chain/go-tss/keysign"
+	"github.com/zeta-chain/go-tss/tss"
```

Comment:
I think we can remove the spaces between the imports and run `make generate` to sort the imports